cli/azd/main.go

```diff
@@ -62,7 +62,9 @@ func main() {
 
 	rootContainer := ioc.NewNestedContainer(nil)
 	ioc.RegisterInstance(rootContainer, ctx)
-	cmdErr := cmd.NewRootCmd(false, nil, rootContainer).ExecuteContext(ctx)
+
+	// Execute command with auto-installation support for extensions
+	cmdErr := cmd.ExecuteWithAutoInstall(ctx, rootContainer)
```

Comment:
How does this feature work with the `alpha.extensions` flag (i.e. when set to `on` vs `off`)?

Reply:
This is currently ignoring the alpha flags!!! XD  

I wonder at what point we will remove the alpha flag from extensions..  and if it is worth it to still keep it as alpha